File: Semester3/ProgrammingParadigms/C1/3/OCaml/main.ml
let value = 5;;
let number = 10;;

let rec replicate(value, number) = (if number <= 0 then [] else value :: replicate(value, number-1));;

let x = replicate(value, number);;

let print_number number = Printf.printf "%d\n" number;;

List.iter print_number x;;

File: Semester3/ProgrammingParadigms/C1/3/Scala/main.scala
object app3 extends App {
    def replicate[T](x : T, number : Int):List[T] = {
        if (number > 0) {
            List(x) ++ replicate(x, number - 1)
        } else Nil
  }


    var value = 10
    var number = 3
  
    var myList = replicate(value, number)

    myList.foreach(element => println(element))

}
